8.1.2/startup/initAppInsights.js

```diff
@@ -0,0 +1,8 @@
+try {
+    // Initialize the Application Insights runtime, using the
+    // instrumentation key provided by the environment.
+    require("applicationinsights").setup().start();
```

Comment:
Also - what does this accomplish for your app if you *don't* have it baked in? I'm assuming it tracks basic metrics?